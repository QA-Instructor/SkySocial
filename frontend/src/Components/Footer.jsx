import React from 'react';
import "../resources/footer.css" 

const Footer = () => {
  return (
    <footer className="footer-container">
      &copy; SKY Ltd 2024-2024
    </footer>
  );
};

export default Footer;
