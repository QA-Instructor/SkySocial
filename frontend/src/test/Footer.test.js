import React from 'react';
import { create } from 'react-test-renderer';
import Footer from '../Components/Footer';

describe('Footer Snapshot tests',()=>{
    test(`Footer matches snapshot`, () => {
        const footer = create(<Footer />);
        expect(footer.toJSON()).toMatchSnapshot();
    });  
});

describe("copyright year appears as expected", () => {
    test(`copyright appears`, () => {
        const testRenderer = create(<Footer />);
        const testInstance = testRenderer.root;
        const copyright = testInstance.findByProps({className:'footer-container'});
        expect(copyright.children).toContain('Â© SKY Ltd 2024-2024');
    })
});