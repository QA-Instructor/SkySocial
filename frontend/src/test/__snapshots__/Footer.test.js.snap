// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot tests Footer matches snapshot 1`] = `
<footer
  className="footer-container"
>
  Â© SKY Ltd 2024-2024
</footer>
`;
